ui/lib/core/addon/components/shamir/dr-token-flow.hbs

```diff
@@ -3,129 +3,130 @@
+    <div class="has-bottom-margin-m">
+      <p>
+        Updating or promoting this cluster requires an operation token, generated by inputting the root key shares. If you'd
+        like to first encrypt the token with a PGP Key, click "Encrypt with PGP key" below, otherwise we can begin generation
+        of the operation token.
       </p>
-    </Shamir::Form>
-  {{else if this.generateWithPGP}}
-    <ChoosePgpKeyForm
-      @onCancel={{fn (mut this.generateWithPGP) false}}
-      @onSubmit={{this.usePgpKey}}
-      @formText={{this.pgpText.form}}
-      @confirmText={{this.pgpText.confirm}}
-      @buttonText="Generate operation token"
-      data-test-dr-token-flow-step="choose-pgp"
-    />
-  {{else}}
-    {{! Generate token flow not started }}
-    <form
-      {{on "submit" this.startGenerate}}
-      id="shamir"
-      aria-label="shamir generate form"
-      data-test-dr-token-flow-step="begin"
-    >
-      <MessageError @errors={{this.errors}} />
-      <div class="has-bottom-margin-m" data-test-shamir-modal-body>
-        <p>
-          Updating or promoting this cluster requires an operation token, generated by inputting the root key shares. If
-          you'd like to first encrypt the token with a PGP Key, click "Encrypt with PGP key" below, otherwise we can begin
-          generation of the operation token.
-        </p>
+    </div>
+    <div class="field is-grouped is-flex-center">
+      <div class="control is-flex-row">
+        <Hds::Button
+          @color="tertiary"
+          @icon="key"
+          {{on "click" (fn (mut this.generateWithPGP) true)}}
+          data-test-use-pgp-key-cta
+          @text="Provide PGP Key"
+        />
+      </div>
+      <div class="control">
+        <span class="has-side-padding-s">
+          or
+        </span>
       </div>
-      <div class="field is-grouped is-flex-center">
-        <div class="control is-flex-row">
-          <button type="button" class="link" {{on "click" (fn (mut this.generateWithPGP) true)}} data-test-use-pgp-key-cta>
-            Provide PGP Key
-          </button>
-        </div>
-        <div class="control">
-          <span class="has-side-padding-s">
-            or
-          </span>
-        </div>
-        <div class="control">
-          <button type="submit" class="button is-primary" data-test-generate-token-cta>
-            Generate operation token
-          </button>
-        </div>
+      <div class="control">
+        <Hds::Button type="submit" data-test-generate-token-cta @text="Generate operation token" />
       </div>
-    </form>
-  {{/if}}
-</section>
-<footer class="modal-card-foot modal-card-foot-outlined">
-  <button type="button" class="button is-secondary" {{on "click" this.onCancelClose}} data-test-shamir-modal-cancel-button>
-    {{if this.encodedToken "Close" "Cancel"}}
-  </button>
-</footer>
\ No newline at end of file
+    </div>
+  </form>
+{{/if}}
+<hr class="has-background-gray-100" />
```

Comment:
Using this as a separator because forms inside modals render their own buttons (instead of being located in the `<M.Footer>`